src/runtime/HalideRuntimeHexagonHost.h

```diff
@@ -43,6 +43,17 @@ extern void *halide_hexagon_detach_device_handle(void *user_context, struct buff
 extern void *halide_hexagon_get_device_handle(void *user_context, struct buffer_t *buf);
 extern uint64_t halide_hexagon_get_device_size(void *user_context, struct buffer_t *buf);
 
+/** Power HVX on and off. Calling a Halide pipeline will do this
+ * automatically on each pipeline invocation; however, it costs a
+ * small but possibly significant amount of time for short running
+ * pipelines. To avoid this cost, HVX can be powered on prior to
+ * running several pipelines, and powered off afterwards. If HVX is
+ * powered on, subsequent calls to power HVX on will be cheap. */
+// @{
+extern int halide_hexagon_power_hvx_on(void *user_context);
```

Comment:
Nit: can we keep the API space smaller by making this a single call with a boolean argument?

    extern int halide_hexagon_power_hvx(void *, bool on);